oci/tests/integration/Makefile

```diff
@@ -6,14 +6,14 @@ TEST_IMG ?= fluxcd/testapp:test
 
 .PHONY: app
 app:
-	CGO_ENABLED=0 go build -v -o app ./testapp
+	CGO_ENABLED=0 GOOS=linux go build -v -o app ./testapp
 
 docker-build: app
-	docker build -t $(TEST_IMG) --load .
+	docker build -t $(TEST_IMG) .
```

Comment:
We need `--load` because I believe most of the developer machines and CI setup use docker buildx which requires this flag to copy the image from the builder to the local image store. Can you move this to a variable, like we do in kustomize-controller and helm-controller, refer https://github.com/fluxcd/helm-controller/blob/1758df2b5b314a9940a47e3c8cbb6b1faa45b16d/Makefile#L22-L23 .